### DISCUSSION TITLE
Actividad 0 - Marocchi Daniel

### DISCUSSION THREAD

**Discussion by DanielUTN** [frlp-utn-ingsoft/catedra#76] [Show and tell]
https://github.com/DanielUTN/Actividad0-Marocchi-Daniel

**Comment by RodrigoJacznik** [comment]
Buenas,
Los commits y sus mensajes están bien.
La app funciona pero la encuesta que esta cargada no tiene opciones. Te olvidaste de pushear o te olvidaste de cargar las opciones?

**Reply by DanielUTN** [reply]
![2024-04-16_193155-1](https://github.com/frlp-utn-ingsoft/catedra/assets/108563375/db4a3a64-46b9-4828-b99a-86f58dcc824b)
![2024-04-16_193218-2](https://github.com/frlp-utn-ingsoft/catedra/assets/108563375/62ed641b-a893-478b-bee2-e97071633051)

Profesor, acabo de hacer modificaciones. Tenia que completar la parte de django shell del Tutorial 2(que ayer lo vimos en clase). Tuve modificar la BD que afecto los modelos, cambios en polls/models.py tambien. Probe la ejecucion y ahora me deja votar.
Gracias,
Saludos

**Reply by ivanduva** [reply]
Perfecto Daniel, queda aprobado
Saludos
